tests/test_failure_propagator.py

```diff
@@ -15,12 +15,14 @@
     pre_ocp_upgrade,
     post_ocp_upgrade,
     post_ocs_upgrade,
+    purple_squad,
     workloads,
     performance,
     scale,
 )
 
 
+@purple_squad
```

Comment:
Shouldn't be the marker for the `test_report_skip_triggering_test` added automatically during the test execution based on which test probably caused the ceph healt issue? @ebenahar

Reply:
Yeah marking this as `purple_squad` was really just to make the `--collectonly` test happy but after talking with Petr i'm going to introduce another test marker that we can use to ignore tests more directly. I can also update the test logic to mark this with the squad based on the failure.

Reply:
I added a new marker: `ignore_owner` which we can apply to test cases to ignore them during the collection squad owner check. I've applied this to `test_failure_propagator`. We can update the logic in the test to apply the proper squad markers based on test errors in another PR to keep this one strictly focused on ownership.